docs/components/motor/_index.md

```diff
@@ -628,6 +628,13 @@ You can find additional assistance in the [Troubleshooting section](/appendix/tr
 
 You can also ask questions on the [Viam Community Slack](https://join.slack.com/t/viamrobotics/shared_invite/zt-1f5xf1qk5-TECJc1MIY1MW0d6ZCg~Wnw) and we will be happy to help.
 
+## Related Services
+
+{{< cards >}}
+{{% card link="/services/data/" class="small" %}}
```

Comment:
It seems like data doesn’t make sense on some of these that aren’t sensors/cameras?